packages/lix-sdk/src/database/json.example.ts

```diff
@@ -0,0 +1,162 @@
+// This is an example file showing how to use the JSON utilities
+// with the Kysely database library
+
+import { Kysely } from "kysely";
+import { jsonb, jsonObjectFrom, jsonArrayFrom } from "./json.js";
+import type { LixDatabaseSchema } from "./schema.js";
+
+// Example of inserting with JSON utilities
+async function exampleInsertWithJsonb(db: Kysely<LixDatabaseSchema>) {
+	// Create a snapshot with JSON content
+	await db
+		.insertInto("snapshot")
+		.values({
+			content: jsonb({
+				key: "value",
+				nested: {
+					property: true,
+					count: 42,
+				},
+			}),
+		})
+		.execute();
+
+	// Insert metadata as JSON object for file
+	await db
+		.insertInto("file")
+		.values({
+			path: "/example/path.txt",
+			data: Buffer.from("Hello World"), // data as binary
```

Comment:
use `new TextEncoder().encode("Hello World")`. `Buffer` is a node api

Reply:
done 👍